### ISSUE TITLE
Option to have free text tag entry

### ISSUE THREAD

**Issue by gingerbeardman** [acez/bookmark-companion#29]
- would be appreciated as it would be quicker to enter tags

**Comment by acez** [comment]
Good point, I am also not completely happy with the way it's working currently. But e.g. my workflow for iPad tries to have as little keyboard input as possible but rather using the apple pencil. But maybe there is a possibility to satisfy both needs. 

I have a vague idea of a "chips"-list with some sort of autocomplete on it with an additional button to add existing tags (like it is working at the moment). But not completely sure yet how to visualize which tags will be created and which ones not. I will keep this task updated with news as soon as I have some. :-)
